web/src/stream_types.ts

```diff
@@ -10,6 +10,9 @@ export const enum StreamPostPolicy {
 }
 
 export type stream_permission_group_setting_type = "can_remove_subscribers_group";
+export const stream_permission_group_settings: stream_permission_group_setting_type[] = [
+    "can_remove_subscribers_group",
```

Comment:
Can we just create such a list using keys of `realm.server_supported_permission_settings.stream` object?